lib/streamlit/elements/widgets/button.py

```diff
@@ -406,6 +476,28 @@ def _download_button(
         self.dg._enqueue("download_button", download_button_proto)
         return button_state.value
 
+    def _link_button(
+        self,
+        label: str,
+        url: str,
+        help: Optional[str],
```

Comment:
nit: I think this needs to be moved to keyword-only arguments (see the spec), or?  

Edit: Ahh, just saw that it's correct in the actual API method... so not sure if this is even relevant